Core/src/org/sleuthkit/autopsy/datamodel/Accounts.java

```diff
@@ -77,11 +78,16 @@
 public class Accounts extends Observable implements AutopsyVisitableItem {
```

Comment:
Is there anything stopping us from putting this class in a datamodel.accounts package that we can then exclude from public API export? Would be get a lot of warnings about exporting a non-public type from the Visitors code?